apps/examples/elf/micomapp/Makefile

```diff
@@ -92,6 +92,7 @@ COMPRESSION_TYPE = 0
 BLOCK_SIZE = 0
 endif
 
+LDELFFLAGS += --gc-sections
```

Comment:
Is it possible to add this in AppLibs.mk so that redundancy can be reduced.

Reply:
yeah, done